flyway-gradle-plugin/src/test/groovy/org/flywaydb/gradle/DefaultFlywayCallback.groovy

```diff
@@ -83,12 +83,12 @@ class DefaultFlywayCallback implements FlywayCallback {
 		LOG.info("afterBaseline")
 	}
 
-	@Override
+	//@Override
 	public void beforeInit(Connection dataConnection) {
 		LOG.info("beforeInit")
 	}
 
-	@Override
+	//@Override
```

Comment:
the interface FlywayCallback.java no these method definitions, so should remove the @override from the two methods